client/src/layouts/BaseLayout.scss

```diff
@@ -2,7 +2,9 @@
 @import 'styles/helpers';
 
 body {
-  background-color: $color-primary;
+  background-image: url('../images/backgound-light-grey.png');
+  background-repeat: repeat-x repeat-y;
```

Comment:
`repeat-x repeat-y` is a wrong property value. The correct one is: `background-repeat: repeat.` And you can even combine it with other `background-*` properties:

```css
// This...
background-image: url(...);
background-repeat: repeat;

// ...is the same as
background: url(...) repeat;
```

But by the way, `repeat` is the default so you can just remove this line.